src/config.rs

```diff
@@ -0,0 +1,67 @@
+use confy;
+use serde::{Serialize, Deserialize};
+use std::io::{self, Write};
+
+const APP_NAME: &str = "lmz";
+const CONFIG_NAME: &str = "config";
+
+#[derive(Default, Debug, Serialize, Deserialize)]
+pub struct Config {
+    pub client_id: String,
+    pub client_secret: String,
+    pub username: String,
+    pub password: String,
+    pub access_token: String,
+}
+
+pub fn path() -> String {
+    let path = confy::get_configuration_file_path(APP_NAME, CONFIG_NAME).expect("expected path");
+    let path_string = String::from(path.to_str().unwrap());
+
+    return path_string;
+}
+
+pub fn read() -> Result<Config, confy::ConfyError> {
+    let cfg: Result<Config, confy::ConfyError> = confy::load(APP_NAME, CONFIG_NAME);
+
+    return cfg;
+}
+
+pub fn configure() {
+    let username = gets("username: ");
+    let password = gets("password: ");
+    let client_id = gets("client_id: ");
+    let client_secret = gets("client_secret: ");
+
+    let cfg: Config = Config {
+        client_id,
+        client_secret,
+        username,
+        password,
+        access_token: String::new(),
+    };
```

Comment:
Don't do this here since it's just one field but you want to instantiate multiple "empty" fields at once (and assuming they all `impl Default`), you can use the `..Default::default()` shorthand to do several `::new()` calls at once for all unspecified fields:
```suggestion
    let cfg: Config = Config {
        client_id,
        client_secret,
        username,
        password,
        ..Default::default()
    };
```

Protip: don't do this inside of an `impl Default` block or you will cause a stack overflow:

```
impl Default for AccessToken {
    fn default() -> Self {
        Self { ..Default::default() }
    }
}
```